optbinning/scorecard/plots.py

```diff
@@ -0,0 +1,289 @@
+"""
+Modules with plots for visualizing model performance.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.metrics import roc_curve, roc_auc_score
+
+
+def plot_auroc(
+    df,
+    target_col,
+    proba_col,
+    title='ROC curve',
+    xlabel='False positive rate',
+    ylabel='True positive rate',
+    roc_color='blue',
+    reference_color='black',
+    savefig=False,
+    figname='auroc.png',
+):
+    """Plot AUROC for dataframe with probabilities and binary target.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        Dataframe with the columns to be plotted.
+
+    target_col : str
+        Column with binary target.
+
+    proba_col : str
+        Column with the calculated probabilites.
+
+    title : str (default='ROC curve')
+        Title for plot.
+
+    xlabel : str (default='False positive rate')
+        Label for the x-axis.
+
+    ylabel : str (default='True positive rate')
+        Label for the y-axis.
+
+    roc_color : str (default='blue')
+        Color for the roc line.
+
+    reference_color : str (default='red')
+        Color for reference line.
+
+    savefig : bool (default=False)
+        To save figure.
+
+    figname : str (default='auroc.fig')
+        Name for figure file.
+
+    Returns
+    -------
+    auroc : float
+        Calculate AUROC score.
```

Comment:
roc_auc_score: float
     Compute Area Under the Receiver Operating Characteristic Curve (ROC AUC) from prediction scores.

Reply:
ok.